docpages/03_example_programs.md

```diff
@@ -370,7 +370,7 @@ int main(int argc, char const *argv[])
 	 * You may use ffmpeg to encode songs to ogg opus:
 	 * ffmpeg -i /path/to/song -c:a libopus -ar 48000 -ac 2 -vn -b:a 96K /path/to/opus.ogg 
 	 */
-	dpp::cluster bot("token");
+	dpp::cluster bot("token", dpp::i_default_intents | dpp::i_message_content);
```

Comment:
these examples need to be rewritten as slash commands. we shouldn't be giving examples which have message based commands except where absolutely necessary.

Reply:
thats a quite complex change. it would rather change it in another pr :)